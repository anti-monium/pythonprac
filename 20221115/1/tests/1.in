@objcount
class C:
	def __init__(self):
		print('Hello!')


c = C()
d, e = C(), C()
print(C.counter)
c = 100500
print(C.counter)
f = C()
del e
print(C.counter)
