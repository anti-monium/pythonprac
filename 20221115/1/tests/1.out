Hello!
Hello!
Hello!
3
2
Hello!
2
